docker/Dockerfile

```diff
@@ -46,6 +49,14 @@ RUN curl https://phar.phpunit.de/phpunit-5.7.5.phar -L -o phpunit.phar \
 	&& chmod +x phpunit.phar \
 	&& mv phpunit.phar /usr/local/bin/phpunit
 
+# Install Webgrind
+RUN curl https://github.com/jokkedk/webgrind/archive/master.zip -L -o /tmp/webgrind.zip \
```

Comment:
should prolly be version pinned to tag or commit instead of just grabbing `master`